### ISSUE TITLE
About PyTorch  Geometric 2.3.1

### ISSUE THREAD

**Issue by Sniper1-love** [LUOyk1999/DAGformer#2]
when I install 2.3.1 version of PyTorch  Geometric I found this error "AttributeError: module 'torch' has no attribute 'sparse_csc'"
So I try the 2.2.0 version of PyTorch Geometric, and then there is no error. So what's the version you choose?

**Comment by Sniper1-love** [comment]
And is 2.2.0 version of PyTorch Geometric ok to use?

**Comment by LUOyk1999** [comment]
Yes, I apologize for the late reply due to work reasons. The 2.2.0 version of PyTorch Geometric is okay. The issue may be related to the versions of torch or cuda.
